### ISSUE TITLE
Moment.js could not be found

### ISSUE THREAD

**Issue by SteckA** [jmsalash/meteor-chartjs#1]
I realize this project has not been updated for a while, but I'll ask anyways:

I've added jmsalash:chartjs and get "Chart.js - Moment.js could not be found! You must include it before Chart.js to use the time scale. Download at http://momentjs.com/"

I've seen you mention moment.js not loading in other github issues and wondered if there is a fix for this. I've tried adding the moment package through npm and atmosphere but I get the same error.

**Comment by luigipacheco** [comment]
Same issue, where you able to figure this out?

**Comment by jmsalash** [comment]
Sorry guys - I forgot to reply to the first comment. This was a basic module I did to add chartjs 2.0 beta to my Meteor webapp at the time when Meteor didn't work very well with npm packages.

I can recall having issues with momentjs, and I believe I solved them by installing the npm package instead using the Meteor module, but not sure. Try also checking the order how momentjs is loaded in Meteor (off the top of my head... these are just suggestions)

In any case, I would recommend you using the npm package instead - it is the suggested approach in the chartjs github and Meteor works great with npm packages now.
